src/test/mir-opt/unreachable.rs

```diff
@@ -0,0 +1,32 @@
+use std::mem::transmute;
```

Comment:
I think if you add 

```
// run-fail
```

to the top of this file, it will allow the test to pass even if running the executable fails.